projects/plugins/jetpack/modules/protect/blocked-login-page.php

```diff
@@ -10,15 +10,42 @@
  * is available, or during the login_head hook.
  *
  * Class will only be instanciated if Protect has detected a hard blocked IP address.
- *
- *
  */
 class Jetpack_Protect_Blocked_Login_Page {
 
-	private static $__instance = null;
+	/**
+	 * Instance of the class.
+	 *
+	 * @var Jetpack_Protect_Blocked_Login_Page
+	 */
+	private static $instance = null;
+
+	/**
+	 * Can send recovery emails.
+	 *
+	 * @var bool - defaults to true.
```

Comment:
I think I would leave the comment as part of the description; it may play better with IDEs.

```suggestion
	 * Can send recovery emails. defaults to true.
	 *
	 * @var bool
```